dockerfile_build/Dockerfile.ubuntu_build

```diff
@@ -2,6 +2,8 @@ FROM bazel/reproducible:ubuntu
 RUN apt-get update -y && \
     apt-get install --no-install-recommends -y -q \
       build-essential \
+      curl \
+      ca-certificates \
```

Comment:
We should get this in the base image. Can you open a bug to track that for now?

Reply:
What base image are you referring to?  This is the 'ubunut+build_essential, etc.' base image.  Are you referring to the 'ubuntu' image?

Reply:
ubuntu

Reply:
Issue here:
https://github.com/GoogleCloudPlatform/debian-docker/issues/117